docs/ref/csrf.txt

```diff
@@ -265,6 +269,12 @@ dangerous side effects , and so a CSRF attack with a GET request ought to be
 harmless. :rfc:`2616` defines POST, PUT and DELETE as 'unsafe', and all other
 methods are assumed to be unsafe, for maximum protection.
 
+The protection assumes an attacker cannot conduct a man-in-the-middle attack,
```

Comment:
I would reword it to something like "the csrf protection cannot protect against man-in-the-middle attackes, …"